packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -309,6 +310,9 @@ export class SingleAddressWallet implements ObservableWallet {
       confirmed$: this.transactions.outgoing.confirmed$,
       logger: contextLogger(this.#logger, 'transactionsReemitter'),
       rollback$: this.transactions.rollback$,
+      stabilityWindowSlotsCount$: this.genesisParameters$.pipe(
+        map((params) => calculateStabilityWindowSlotsCount(params))
```

Comment:
Since it's not used anywhere else in SingleAddressWallet, I think it would be slightly better to move this .pipe(map) to createTransactionReemitter (passing just the necessary genesisParameters$ to it).

Reply:
On the other hand, doing this passes unnecessary data to TransactionReemiter, instead of just the bare minimum it needs.

Reply:
c66f5797594c56d155aeb62836e4228622916a55